winpr/libwinpr/sspi/Kerberos/kerberos.c

```diff
@@ -643,7 +656,11 @@ static SECURITY_STATUS SEC_ENTRY kerberos_EncryptMessage(PCtxtHandle phContext,
 	sspi_gss_buffer_desc input;
 	sspi_gss_buffer_desc output;
 	PSecBuffer data_buffer = NULL;
+	PSecBuffer sig_buffer = NULL;
 	context = (KRB_CONTEXT*)sspi_SecureHandleGetLowerPointer(phContext);
+	BYTE *ptr;
```

Comment:
Better move that above `context = (KRB_CONTEXT*)sspi_SecureHandleGetLowerPointer(phContext);`, VS2010 is nasty...

Reply:
done